src/fabric.api.ts

```diff
@@ -50,6 +51,7 @@ export interface FabricApi {
         host?: string,
         port?: number,
         endpoint?: string,
+        useSSL?: boolean,
```

Comment:
This means this is a breaking change as it introduces the optional arg in the middle of the signature. We will have to bump a feature version.

Reply:
actually that's a good point. i wanted to group arguments by similar/same areas but that essentially breaks the existing API.